everviz/pages/controls.py

```diff
@@ -0,0 +1,50 @@
+from pandas import DataFrame
+
+
+def control_data_per_batch(api):
+    return DataFrame(api.control_values)
+
+
+def control_data_initial_vs_best(api):
+    objectives_df = DataFrame(api.objective_values)
+    best_batch = objectives_df.batch[objectives_df.value.idxmax()]
+    data = DataFrame(api.control_values)
+
+    # Keep only controls associated with initial and best batches
+    data = data[(data.batch == 0) | (data.batch == best_batch)]
+    return data.replace({"batch": 0}, "initial").replace({"batch": best_batch}, "best")
+
+
+def page_layout(controls_per_batch, controls_initial_vs_best):
+    return {
+        "title": "Controls",
+        "content": [
+            "## Control value per batch",
```

Comment:
Is there a reason for having 2 #?

Reply:
I found that having just one `#` which is H1 in markdown results in a title text which is too big for individual plot titles.

Reply:
My bad, I was sleeping, its not a comment marker :smile: